service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -156,6 +159,11 @@ private void addIncomingReferral(Bundle bundle, Encounter encounter) {
 
     }
 
+    private void addCarePlan(Bundle bundle, List<CarePlan> carePlans) {
+        if (!carePlans.isEmpty())
```

Comment:
I don't think you need this check - if it's empty stream will not iterate over it. Other question - can it be null?

Reply:
Removed the empty check accordingly